src/algebra/monoid_algebra/support.lean

```diff
@@ -13,7 +13,55 @@ universes u₁ u₂ u₃
 namespace monoid_algebra
 
 open finset finsupp
-variables {k : Type u₁} {G : Type u₂} {R : Type u₃} [semiring k]
+variables {k : Type u₁} {G : Type u₂} [semiring k]
+
+lemma support_single_mul_subset [decidable_eq G] [has_mul G]
```

Comment:
Should `r` and `a` be explicit here (and below)?

Reply:
I changed these implicit arguments to explicit, thanks!

Reply:
I'll wait for CI to pass before merging.